injitest/androidRegression.xml

```diff
@@ -96,6 +96,7 @@
                 <methods>
                     <include name="noCardsAvailableToShare"/>
                     <include name="verifyReceivedCardTabPresent"/>
+                    <include name="verifyPopupsForShareWithSelfi"/>
```

Comment:
spelling typo mistake verifyPopupsForShareWithSelfi should be `verifyPopupsForShareWithSelfie`